City-of-Helsinki/paatokset-search
# Pp 238 policymakers search

Added  search functionality for policymakers data. 

To test: 

- Go to backend branch (https://github.com/City-of-Helsinki/helsinki-paatokset/pull/89) and do actions listed there
- Run `npm start`. Make sure your `.env` has `REACT_APP_ELASTIC_URL` that points to your elastic endpoint (`localhost:9200`) should work
- Open `public/index.html` in an editor. Change the `data-type` -attribute to "policymakers".
- Open up localhost:3000 in your browser. Try some searches out.
- Try out keyboard controls (is the app accessible?)
